packages/openapi/definitions/decorators.ts

```diff
@@ -0,0 +1,68 @@
+import { DecoratorContext, Model, Operation, Type, TypeSpecValue } from "@typespec/compiler";
```

Comment:
do we want that file commited?

any better idea for the folder name it should be generated in?